proposals/0048-precompile-for-secp256r1-sigverify.md

```diff
@@ -0,0 +1,277 @@
+---
+simd: "0048"
```

Comment:
@iceomatic since SIMD-48 is not living, this would actually be another SIMD-75 that "supersedes" SIMD-48. I'd recommend updating the headers to display this.

Reply:
Just out of curiosity, how come it's not considered "living"? The original was merged a while ago and we've been working on it since then.

Reply:
It's to avoid continually updating a single SIMD, or give the impression that "we can accept now and change everything later" leading to less rigorous review. The [living](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0001-simd-process.md#living) status is only for specific SIMDs that could update a decent amount, like simd-1 or simd-7.

Reply:
Ah totally fair, makes sense. Will amend, thx 🫡

Reply:
Resolved 👍